subsys/dfu/src/dfu_target_mcuboot.c

```diff
@@ -75,6 +109,31 @@ int dfu_target_mcuboot_init(size_t file_size)
 {
 	int err = flash_img_init(&flash_img);
 
+	if (IS_ENABLED(CONFIG_SETTINGS_FCB)) {
+		err = settings_subsys_init();
+		static struct settings_handler sh = {
```

Comment:
Having this declaration here looks weird, could we move it above the subsys_init ?

Reply:
Yeah, could do it. I don't know why it's below I think it was copypasta from another project.